parsers/US_PJM.py

```diff
@@ -0,0 +1,179 @@
+# The arrow library is used to handle datetimes
```

Comment:
Are you sure you want the parser to be part of this pull request?
I will only be able to merge once the parser works.

Reply:
@corradio - I'm working on 4 or 5 parsers/geographies at once.  That US_PJM.py commit went here is my mistake (I really don't know how to use Github - Systemcatch is helping me a bit on Slack, but it is honestly an unfamiliar system/language to me and I need "how to do it" explained in "bubba speak" - another way of saying for a Github idiot [me])

We need "issues" or "Pull Requests" open for each of these areas, whether you prefer to refer to them as Parsers (as the main driver) or "geographies" doesn't matter to me.  So at minimum [again I am an idiot in Github] we can attribute commits accordingly.

For North America (at present time), we need 1 for each of the following...
- Ontario (maybe within Canada is ok as it is substantially finished)
- Quebec
- NE-ISO (New England) - already open
- NYISO (New York) - already open, but a dumping ground for PJM too
- PJM (Penn, NJ, Ohio, etc) - currently being dumped here
- MISO (Michigan, Wisconsin, Indiana, Minnesota, Manitoba (also), others)
- TVA (Tennessee)
- ERCOT (Texas)
- SwPP (Prairie States)

That should suffice for now.  I know that I can commit accordingly (and focused) this way, but without open (whatever they are - issues or PRs) we'll never close them with cross mingled dumping grounds like "add Canada" or "add USA".

If you as admin can back or delete the US_PJM.py commit, that would be great!  I would highly appreciate it! :)

Reply:
Unfortunately there's absolutely nothing I can do.
A pull request is a request to the main repository (tmrowco/electricitymap/master) to pull the changes from your fork (hanshyde/electricitymap/master), so I have no power to modify anything.
I suggest that you separate your changes in branches (hanshyde/electricitymap/pjm-capacity etc...) and that you submit a separate pull request for each instead of submitting one from your master branch.
Pragmatically, you can create a branch for the parser, and move it to there instead.